tests/unit/components/video-box/VideoBox.spec.jsx

```diff
@@ -0,0 +1,25 @@
+import { screen, render } from '@testing-library/react'
+import VideoBox from '~/components/video-box/VideoBox'
+
+describe('VideoBox component test', () => {
+  const video = '~/assets/img/guest-home-page/videoImg.png'
```

Comment:
For what purpose you use this constant? On 22th line you shadow it.

Reply:
I've just remove this constant